SpatialBinaryLogisticRegression.lua

```diff
@@ -0,0 +1,84 @@
+------------------------------------------------------------------------
+--[[ SpatialBinaryLogisticRegression ]]--
+-- Takes an image of size batchSize x nChannel x width x height as input.
+-- Computes Binary Logistic Regression Cost.
+-- Useful for 2 class pixel classification.
+------------------------------------------------------------------------
+
+local SpatialBinaryLogisticRegression, parent = torch.class('nn.SpatialBinaryLogisticRegression', 'nn.Criterion')
+
+function SpatialBinaryLogisticRegression:__init(sizeAverage)
```

Comment:
can you remove sizeAverage from constructor argument. Just make it `self.sizeAverage = true`. If the user wants to change that, he can do so by setting it via 
```lua
criterion = SpatialBinaryLogisiticRegression() 
criterion.sizeAverage = false
```
This is standard practice for nn.